.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,28 @@
+name: "Publish to GitHub Pages"
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows:
+      - "Build and Export Game"
+    types:
+      - completed
+
+# Cancel any ongoing previous run if the job is re-triggered
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  publish:
+    name: Publish all branches to GitHub Pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: endlessm/amalgamate-pages@v1
```

Comment:
Side note: I see a mention to "Everlasting Candy" in https://github.com/endlessm/amalgamate-pages?tab=readme-ov-file#example . The link redirects fine to the new renamed repo, though.